src/Blazored.Modal/BlazoredModalInstance.razor.cs

```diff
@@ -28,7 +28,7 @@ public partial class BlazoredModalInstance : IDisposable
 
     [SuppressMessage("Style", "IDE0044:Add readonly modifier", Justification = "This is assigned in Razor code and isn't currently picked up by the tooling.")]
     private ElementReference _modalReference;
-    private FocusTrap _focusTrap = default!;
+    public FocusTrap? FocusTrap = default;
```

Comment:
If we're making this public I think it would be better to swap to a property. As it's now nullable, we can remove the default assignment as well. Could you also move this a few lines up so it's with the other properties?

```suggestion
    public FocusTrap? FocusTrap { get; set; }
```

Reply:
Hi Chris,

Sure, changes are in now.